proto/p4/p4runtime.proto

```diff
@@ -255,10 +255,10 @@ message MeterEntry {
 // meters is not allowed and will return an error. The insertion/deletion
 // should happen as part of INSERT/DELETE on the associated table-entry.
 // For ReadRequest, the scope is defined as follows:
```

Comment:
The comment only mentions `ReadRequest` but in the code you seem to imply that wildcards will also be supported in the long-term for `WriteRequest`. I am not opposed to this, but I think the comment here should mention `WriteRequest` as well if we intend to support this eventually.

Reply:
Removed wildcard for WriteRequest. I don't know any use case for it. Treating missing table_entry as INVALID_ARGUMENT

Reply:
so to summarize:
- for a WriteRequest: table_entry is required, table_entry.table_id must be valid and the match key must be provided
- for a ReadRequest: default table_entry means "read-all entries in all direct counters / meters". Otherwise table_entry.table_id must be valid; in which case a missing match key means "read-all entries in the direct counter / meter for this specific table" and a valid match key means "read just this specific entry in the direct counter / meter for this specific table".

Please confirm.